src/main/drivers/pwm_output.c

```diff
@@ -135,7 +136,9 @@ static void pwmWriteMultiShot(uint8_t index, uint16_t value)
 
 void pwmWriteMotor(uint8_t index, uint16_t value)
 {
-    pwmWritePtr(index, value);
+    if (motors[index].enabled) {
+        pwmWritePtr(index, value);
+    }
```

Comment:
It would be good if that, rather than checking here, there was a check at a higher level. At high looptimes (eg 16kHz or greater) this check is significant.